notebooks/QA_evaluation_metrics.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>The pdf faq is not exactly the same as rosa workshop docs, right? Should we use the md doc only?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
> The pdf faq is not exactly the same as rosa workshop docs, right? 

Correct, they are not fully equal at the moment - although my understanding is that they will/should eventually be the same.

> Should we use the md doc only?

IMO, yes:

- we don't really care if the questions are exactly the same, we only want a set of questions and answers (plus they are quite similar already)
- with #19 we already get a dataset of QA, so no need to parse another one